server/security.js

```diff
@@ -9,7 +9,11 @@ const defaultTrustedDomains = {
   connection: [ "'self'" ],
   frame: [
           "'self'",
-          "https://docs.google.com"],
+          "https://docs.google.com"
+         ],
+  iframe: [
```

Comment:
I think you want to allow pontoon on `script` part as well. Pretty much you want to make sure Pontoon can do everything like it's an internal server? https://github.com/mozilla/donate.mozilla.org/blob/master/server.js#L282-L293